### PR TITLE
# Add NSTART token metadata

### PR DESCRIPTION
**Pull request by edeava**
# Add NSTART token metadata 

NEP-141: https://explorer.mainnet.near.org/accounts/near_starter_token.near
Aurora ERC-20 (BlockScout): https://explorer.mainnet.aurora.dev/address/0x06aebb0f3d9ebe9829e1b67bd3dd608f711d3412
Aurora ERC-20 (Aurorascan): https://aurorascan.dev/address/0x06aebb0f3d9ebe9829e1b67bd3dd608f711d3412

### CODE REVIEW

File: tokens/nstart.json [hunk 1]
```diff
@@ -0,0 +1,8 @@
+{
+    "ethereum_address": "",
+    "near_address": "near_starter_token.near",
+    "aurora_address": "0x06aebb0f3d9ebe9829e1b67bd3dd608f711d3412",
+    "name": "Near starter",
+    "symbol": "NSTART",
+    "decimals": 24
```
Comment (sept-en) [inline tokens/nstart.json:7]:
```suggestion
    "decimals": 24,
    "icon": "data:image/svg+xml,%0A%3Csvg width='128' height='128' viewBox='0 0 128 128' fill='none' xmlns='http://www.w3.org/2000/svg'%3E%3Crect width='128' height='128' fill='%23FF85FB'/%3E%3Cpath d='M38.6667 64L26 64L26 76.6667L38.6667 76.6667L38.6667 64Z' fill='%23090909'/%3E%3Cpath d='M102 64L89.3333 64L89.3333 76.6667L102 76.6667L102 64Z' fill='%23090909'/%3E%3Cpath d='M26.0001 89.3334L38.6667 89.3334L38.6667 102L26.0001 102L26.0001 89.3334Z' fill='%23090909'/%3E%3Cpath d='M89.3333 89.3334L102 89.3334L102 102L89.3333 102L89.3333 89.3334Z' fill='%23090909'/%3E%3Cpath d='M26.0001 76.6666L38.6667 76.6666L38.6667 89.3333L26.0001 89.3333L26.0001 76.6666Z' fill='%23090909'/%3E%3Cpath d='M89.3333 76.6666L102 76.6666L102 89.3333L89.3333 89.3333L89.3333 76.6666Z' fill='%23090909'/%3E%3Cpath d='M38.6667 51.3334L26 51.3334L26 64L38.6667 64L38.6667 51.3334Z' fill='%23090909'/%3E%3Cpath d='M102 51.3334L89.3333 51.3334L89.3333 64L102 64L102 51.3334Z' fill='%23090909'/%3E%3Cpath d='M38.6667 38.6666L26 38.6666L26 51.3333L38.6667 51.3333L38.6667 38.6666Z' fill='%23090909'/%3E%3Cpath d='M102 38.6666L89.3333 38.6666L89.3333 51.3333L102 51.3333L102 38.6666Z' fill='%23090909'/%3E%3Cpath d='M38.6667 26L26 26L26 38.6667L38.6667 38.6667L38.6667 26Z' fill='%23090909'/%3E%3Cpath d='M102 26L89.3333 26L89.3333 38.6667L102 38.6667L102 26Z' fill='%23090909'/%3E%3Cpath d='M51.3334 38.6666L38.6667 38.6666L38.6667 51.3333L51.3334 51.3333L51.3334 38.6666Z' fill='%23090909'/%3E%3Cpath d='M63.9999 51.3334L51.3333 51.3334L51.3333 64L63.9999 64L63.9999 51.3334Z' fill='%23090909'/%3E%3Cpath d='M76.6667 64L64 64L64 76.6667L76.6667 76.6667L76.6667 64Z' fill='%23090909'/%3E%3Cpath d='M89.3334 76.6666L76.6667 76.6666L76.6667 89.3333L89.3334 89.3333L89.3334 76.6666Z' fill='%23090909'/%3E%3C/svg%3E",
    "reference": "",
    "reference_hash": ""
```

**Review by sept-en — CHANGES_REQUESTED** [review]

**Review by sept-en — APPROVED** [review]

**Comment by sept-en** [comment]
Thanks for your PR. The metadata will be updated in the coming days.

**Comment by AndrewSolovyev** [comment]
@sept-en Hello, for some reason explorer still shows nothing for our token. Can you help please?

**Comment by AndrewSolovyev** [comment]
we still dont have any information about the token on the chain. Please help @sept-en

**Comment by mike-tis** [comment]
Hello gentlemen 
I see that NSTART token is available on the bridge, but as a custom token only. Can we move it to the main tokenlist? This is the official token of nearstarter.fi launchpad. 
Also, we've made one bridging operation and we see the NSTART balance on the Aurora wallet not, but there's no tx... neither in the explorer, nor in the wallet. 
@sept-en please help

**Pull request merged**